jupyter_elfinder/elfinder.py

```diff
@@ -941,24 +961,12 @@ def __cwd(self, path: str) -> None:
             "volumeid": self.volumeid,
         }
 
-    def __cdc(self, path: str) -> None:
-        """Get Current Directory Content."""
-        files = []
-        dirs = []
-
-        for fil in sorted(os.listdir(path)):
-            if not self.__is_accepted(fil):
-                continue
-            file_path = os.path.join(path, fil)
-            info = self.__info(file_path)
-            info["hash"] = self.__hash(file_path)
-            if info["mime"] == "directory":
-                dirs.append(info)
-            else:
-                files.append(info)
+        try:
+            info["dirs"] = any(next(os.walk(path))[1])
+        except StopIteration:
+            info["dirs"] = False
```

Comment:
```suggestion
            info["dirs"] = None
```

Reply:
Right, I just noticed that in the spec, it `dirs` should be a Number, I changed this in a later commit.